x/ibc/transfer/keeper/msg_server_test.go

```diff
@@ -204,6 +204,7 @@ func (suite *KeeperTestSuite) TestTransfer() {
 				pair, err := suite.app.Erc20Keeper.RegisterERC20(suite.ctx, contractAddr)
 				suite.Require().NoError(err)
 				suite.Commit()
+				suite.Require().Equal(pair.Erc20Address, strings.TrimPrefix(pair.Denom, "erc20/"))
```

Comment:
nit: testing this way instead of trimming the prefix is more precise here - if the prefix was missing in `pair.Denom` this test would still pass
```suggestion
				suite.Require().Equal("erc20/"+pair.Erc20Address, pair.Denom)
```